pyproject.toml

```diff
@@ -10,12 +10,13 @@ repository = "https://github.com/djrobstep/migra"
 homepage = "https://databaseci.com/docs/migra"
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4"
+python = ">=3.8,<4"
 sqlbag = "*"
 six = "*"
-# schemainspect = {path="../schemainspect", develop=true}
-schemainspect = ">=3.1.1663480743"
+#schemainspect = { path = "../schemainspect", develop = true }
+schemainspect = { git = "https://github.com/joshainglis/schemainspect.git", rev = "79b3fbdb9c22252574578e8f83dcb7f85ebaf69a" }
```

Comment:
Will need to update this prior to merging if https://github.com/djrobstep/schemainspect/pull/90 gets merged